src/libraries/System.Formats.Nrbf/src/System/Formats/Nrbf/RecordMap.cs

```diff
@@ -56,7 +56,7 @@ internal void Add(SerializationRecord record)
                     return;
                 }
 #endif
-                throw new SerializationException(SR.Format(SR.Serialization_DuplicateSerializationRecordId, record.Id));
+                throw new SerializationException(SR.Format(SR.Serialization_DuplicateSerializationRecordId, record.Id._id));
```

Comment:
`SerializationRecordId` does not override `ToString`, we need to specify the Id in explicit way

Reply:
Should it override ToString?